### ISSUE TITLE
Guidance on `application/ld+json` 

### ISSUE THREAD

**Issue by OR13** [w3c/vc-data-model#1040]
We should beware of the existing behavior for `ld+json` and comment on if this media type is acceptable for `credential+ld+json` and `verifiable-credential+ld+json`

We had a similar challenge with `did+json`, `did+ld+json` and the default behavior for `did.json` in DID Web.

We should anticipate developers lagging behind adopting support for new media types.

**Comment by OR13** [comment]
I suggest we say that developers can do the following:

```html
<script type="application/vc+ld+jwt">{vcLdJwt}</Script>
        <Script type="application/vc+ld+json">
          {JSON.stringify(
            {
              '@context': [
                'https://www.w3.org/ns/credentials/v2',
                'https://w3id.org/vc/status-list/2021/v1',
              ],
              id: 'https://vendor.example/credentials/status/3',
              type: ['VerifiableCredential', 'StatusList2021Credential'],
              issuer: 'did:example:123',
              validFrom: '2021-04-05T14:27:40.000Z',
              credentialSubject: {
                id: 'https://vendor.example/credentials/status/3#list',
                type: 'StatusList2021',
                statusPurpose: 'suspension',
                encodedList: 'H4sIAAAAAAAAA2MAAI3vAtIBAAAA',
              },
            },
            null,
            2,
          )}
        </script>
        <script type="application/ld+json">
          {JSON.stringify(
            {
              '@context': 'https://schema.org',
              '@type': ['EducationalOccupationalCredential'],
              name: 'Certified Safety Professional',
              url: 'https://www.bcsp.org/CSP',
              description: `Certified Safety Professionals (CSP) are persons who 
perform at least 50% of professional level safety duties, 
including making worksite assessments to determine risks, 
assessing potential hazards and controls, 
evaluating risks and hazard control measures, 
investigating incidents, 
maintaining and evaluating incident and loss records, 
and preparing emergency response plans.`,
              credentialCategory: {
                '@type': 'DefinedTerm',
                '@id': 'http://purl.org/ctdl/terms/Certification',
                name: 'Certification',
                inDefinedTermSet: {
                  '@type': 'DefinedTermSet',
                  '@id': 'http://purl.org/ctdl/terms/',
                  name: 'Credential Transparency Description Language',
                },
              },
            },
            null,
            2,
          )}
        </script>
```

**Comment by brentzundel** [comment]
I am not clever enough to understand what you are saying with the above code block and would appreciate if you could dumb it down for me.

**Comment by melvincarvalho** [comment]
I'd like to share an observation regarding the media type used here:

`<script type="application/vc+ld+json">`

Wouldn't it have been more intuitive and developer-friendly, easier to search, match regex, had standards developed in this way:

- application/json
- application/json+ld
- application/json+ld+did

Or even:

- json/ld
- json/ld+did

The use of "application/vc+ld+json" seems unconventional, even to someone experienced in standards. In general, developers tend to be more at ease with "application/ld+json." I recommend using that format if feasible.

This is merely an observation. Who knows, maybe the json/ld MIME type will gain traction in the future.

**Comment by OR13** [comment]
@melvincarvalho I suggest you send your comments regarding structured suffixes to IETF, before it is too late :)

IANA is not going to let us register any of these, unless IETF explains how to process them, and that is not going super well atm... which makes taking a dependency on multiple `+` media types... not advisable.

**Comment by OR13** [comment]
@brentzundel these script tags are processed by search engines as they crawl web pages... those engines are very unlikely to do anything with stuff that is not `application/ld+json`.

See: 

- https://moz.com/blog/json-ld-for-beginners
- https://schema.org/docs/datamodel.html

**Comment by OR13** [comment]
Related open PR: https://github.com/w3c/json-ld-syntax/pull/415

**Comment by msporny** [comment]
PR #1189 has been raised to address this issue. This issue will be closed once that PR has been merged.

**Comment by msporny** [comment]
PR #1189 has been merged, closing.
